From c2fdea8a5c152a81eb8c74df0932f5e79ed3e8c3 Mon Sep 17 00:00:00 2001
From: Gwenn Bourree <gwenn.bourree@intel.com>
Date: Wed, 17 Jun 2015 10:48:54 +0200
Subject: [PATCH] [Telephony/MCD]: Remove potential panic printf

In order to avoid a kernel panic if the gpio
descriptors are NULL, the pr_info command is
removed. This could happen when there is no GPIO
defined (for 7360 modem for instance) and the
build "eng" is selected.

Change-Id: Icc146b780189f7bb4efe44466d4f21d4e22cf191
Signed-off-by: Gwenn Bourree <gwenn.bourree@intel.com>
Tracked-On: https://jira01.devtools.intel.com/browse/GMINL-11461
---
 drivers/staging/modem_control/mcd_cpu.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/drivers/staging/modem_control/mcd_cpu.c b/drivers/staging/modem_control/mcd_cpu.c
index 0f2fef66ed97..99e6733dffce 100644
--- a/drivers/staging/modem_control/mcd_cpu.c
+++ b/drivers/staging/modem_control/mcd_cpu.c
@@ -129,13 +129,6 @@ int cpu_init_gpio(void *data)
 		pr_info("No CORE_DUMP GPIO set");
 	}
 
-	pr_info(DRVNAME
-		": GPIO (rst_bbn: %d, pwr_on: %d, rst_out: %d, fcdp_rb: %d)\n",
-		desc_to_gpio(cpu_data->gpio_rst_bbn),
-		desc_to_gpio(cpu_data->gpio_pwr_on),
-		desc_to_gpio(cpu_data->gpio_rst_out),
-		desc_to_gpio(cpu_data->gpio_cdump));
-
 	return 0;
 
  free_ctx2:
-- 
1.9.1

